test/test_utils_config_module.py

```diff
@@ -137,30 +143,30 @@ def test_codegen_config(self):
         del config.e_ignored
 
     def test_get_hash(self):
+        print(config.get_hash())
         self.assertEqual(
```

Comment:
consider using assertExpectedInline

Reply:
That's cool :D. Updated.

Reply:
Turns out it doesn't work

(working with the raw bytes is fun, but even after converting to hex, it still fails :(

```
 % EXPECTTEST_ACCEPT=1    python test/test_utils_config_module.py TestConfigModule.test_get_hash

Accepting new output for __main__.TestConfigModule.test_get_hash at /home/clr/pytorch/test/test_utils_config_module.py:167
F
======================================================================
FAIL: test_get_hash (__main__.TestConfigModule.test_get_hash)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/clr/pytorch/torch/testing/_internal/common_utils.py", line 2995, in wrapper
    method(*args, **kwargs)
  File "/home/clr/pytorch/test/test_utils_config_module.py", line 167, in test_get_hash
    self.assertExpectedInline(
  File "/home/clr/pytorch/torch/testing/_internal/common_utils.py", line 2941, in assertExpectedInline
    return super().assertExpectedInline(actual if isinstance(actual, str) else str(actual), expect, skip + 1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/clr/.conda/envs/pytorch-deps/lib/python3.12/site-packages/expecttest/__init__.py", line 351, in assertExpectedInline
    assert_expected_inline(
  File "/home/clr/.conda/envs/pytorch-deps/lib/python3.12/site-packages/expecttest/__init__.py", line 292, in assert_expected_inline
    assert old != new, (
           ^^^^^^^^^^
AssertionError: Failed to substitute string at /home/clr/pytorch/test/test_utils_config_module.py:167; did you use triple quotes?  If this is unexpected, please file a bug report at https://github.com/ezyang/expecttest/issues/new with the contents of the source file near /home/clr/pytorch/test/test_utils_config_module.py:167

To execute this test, run the following from the base repo dir:
    python test/test_utils_config_module.py TestConfigModule.test_get_hash

This message can be suppressed by setting PYTORCH_PRINT_REPRO_ON_FAILURE=0

----------------------------------------------------------------------
Ran 1 test in 0.069s

FAILED (failures=1)
```

Reply:
You can't leave me hanging like this, what exactly did you put here lol

Reply:
Repro at https://github.com/pytorch/expecttest/issues/24

Diff is

```
     def test_get_hash(self):
-        self.assertEqual(
-            config.get_hash(), b"U\x8bi\xc2~PY\x98\x18\x9d\xf8<\xe4\xbc%\x0c"
+        self.assertExpectedInline(
+            config.get_hash(), ''
         )
         # Test cached value
         self.assertEqual(
-            config.get_hash(), b"U\x8bi\xc2~PY\x98\x18\x9d\xf8<\xe4\xbc%\x0c"
+            config.get_hash(), ''
         )
         self.assertEqual(
-            config.get_hash(), b"U\x8bi\xc2~PY\x98\x18\x9d\xf8<\xe4\xbc%\x0c"
+            config.get_hash(), ''
         )
         config._hash_digest = "fake"
         self.assertEqual(config.get_hash(), "fake")
 
         config.e_bool = False
         self.assertNotEqual(
-            config.get_hash(), b"U\x8bi\xc2~PY\x98\x18\x9d\xf8<\xe4\xbc%\x0c"
+            config.get_hash(), ''
         )
         config.e_bool = True
 
         # Test ignored values
         config.e_compile_ignored = False
         self.assertEqual(
-            config.get_hash(), b"U\x8bi\xc2~PY\x98\x18\x9d\xf8<\xe4\xbc%\x0c"
+            config.get_hash(), ''
         )
         for k in config._config:
             config._config[k].user_override = _UNSET_SENTINEL
             ```